app.js

```diff
@@ -1,9 +1,81 @@
 const express = require('express');
 const app = express();
-const createMyCustomApi = require('./routes');
+const fs = require('fs');
+
+//const createMyCustomApi = require('./routes');
+const { stringify } = require('querystring');
 const PORT = 9000;
 
-app.use('/api', createMyCustomApi);
+app.use(express.json());
+
+// Endpoint to get a list of all team members (get request)
+app.get('/api/getAllMembers', function(req, res){
+    fs.readFile(__dirname + "/" + "members.json", 'utf8', function(err, data){
+        res.status(200);
+        res.end(data);
+    });
+})
+
+// Endpoint to add a new member to the team (post request)
+app.post('/api/addNewMember', (req, res) => {
+    let newMember = {
+        memberId: req.body.memberId,
+        role: req.body.role,
+        firstname: req.body.firstname,
+        lastname: req.body.lastname
+    }
+    fs.readFile(__dirname + "/" + "members.json", 'utf8', function(err, data){
+        var jsonData = JSON.parse(data);
+        jsonData.push(newMember);
+        var newData = JSON.stringify(jsonData);
+        fs.writeFile(__dirname + "/" + "members.json", newData, err => {
```

Comment:
For this particular task, this solution can be enough.
But to align with the production code we are writing on Portal, could you please do the following exercise
- Nested code is hard to read, can you think how you can flatten it so the flow reads just:
```
const members = loadMembers();
members.push(newMember);
saveMembers(members);
```
- Feel free to apply `Repository` pattern to separate file interactions from API

Reply:
I did it. I don't know much about repository patterns...

Reply:
That's cool!
You'll basically implement it while addressing this comment: https://github.com/Theofilos1997/express-restful-api-project/pull/1#discussion_r1711903371